docs/server/features/archiving_blog_post.md

```diff
@@ -0,0 +1,248 @@
+In KurrentDB, the tail of the transaction log, i.e., the latest chunk files, often naturally represent most of the _hot_ data, while old chunk files often represent most of the _cold_ data.
+Thus, the basic idea is to keep the latest X chunks on the local disk while older chunks are stored in the archive based on a user-defined retention policy.
+
+The user-defined retention policy is simple: Keep X _days_ or Y _bytes_ of chunk data locally, whichever is larger.
+To determine proper values for X & Y, there are some built-in metrics in KurrentDB.
+
+You may also observe that there could be some _hot_ data present in the old chunks.
+This is handled through caching mechanisms to avoid hitting the archive frequently (work is in progress to improve this area).
+
+## The _Archiver_ node
+Using _Archiving_ requires adding a new node, called the _Archiver_ node, to the cluster. This node is solely responsible for uploading chunks to the archive.
+
+An archiver node is like a normal database node, except that it is designated as a read-only replica and as an archiver:
+```
+ReadOnlyReplica: true
+Archiver: true
+```
+
+You might say, "Wait a minute! Adding a new node means using even more disk space, right? And this defeats the purpose as our original objective was to reduce disk space."
+
+The above statement may be valid for a short time when you first start to use archiving.
+But with time, as chunks are uploaded to the archive, disk space will be freed from all the nodes, including the _Archiver_ node.
+
+Usually, the _hot_ data is much smaller than the _cold_ data. Therefore, the disks on the nodes can be kept relatively small, and the space used by the extra node will not be significant.
+Note that there's also only one copy of the data in the archive.
+
+## Demo time!
+We'll use Linux for this demo, but the steps should be quite similar on Windows.
+
+Enabling archiving is simple. Just add a few lines to your KurrentDB configuration file (located at `/etc/kurrentdb/kurrentdb.conf`) on all your nodes, including the _Archiver_ node:
+```
+Licensing:
+  LicenseKey: <your key>
+
+Archive:
+  Enabled: true
+
+  StorageType: S3
+  S3:
+    Region: eu-west-1
+    Bucket: kurrentdb-cluster-123-archive
+
+  RetainAtLeast:
+    Days: 30
+    LogicalBytes: 500000000
+```
+
+The configuration settings are straightforward - we first specify a license key and then enable archiving.
+We then specify the storage type. Currently, we support only Amazon S3.  If you want us to support additional cloud providers, we would be happy to [hear from you](https://esdb.ideas.eventstore.com/)!
+Then, we specify the S3 region and the S3 bucket where chunks will be archived.
+
+Finally, we specify the retention policy.
+Here, it means:
+```
+Keep the last chunks containing data that is not older than 30 days
+or keep the latest 500MB of data, whichever is larger
+```
+We keep roughly two chunks of data or more depending on the data's age.
+
+## S3 Authentication
```

Comment:
It would be good to mention that it'll work if you just log in to the aws cli, no need to adjust the files

Reply:
updated, thanks